thehive-charts/thehive/values.yaml

```
@@ -53,120 +65,127 @@ thehive:
-  # TheHive ElasticSearch configuration
   index:
-    # Credentials to connect to ElasticSearch
-    # They should match the credentials for ElasticSearch subchart below (if enabled)
+    # -- ElasticSearch username for index connection
     username: "elastic"
+    # -- ElasticSearch password for index connection (should match ElasticSearch subchart credentials if enabled)
     password: ""
-    # The name of an existing k8s secret containing ElasticSearch password
+    # -- Name of existing Kubernetes secret containing ElasticSearch password
     k8sSecretName: ""
+    # -- Key in the secret containing ElasticSearch password
     k8sSecretKey: "elasticsearch-password"
-    # List of ElasticSearch hostnames
-    # Auto set to ElasticSearch subchart service if enabled and this list is empty: "{{ .Release.Name }}-elasticsearch"
+    # -- List of ElasticSearch hostnames (auto-configured if ElasticSearch subchart is enabled and this is empty)
     hostnames: []
       #- "myelasticsearch.example.com"
 
-  # TheHive object storage configuration
   storage:
-    # Credentials to connect to the object storage
-    # They should match the credentials for MinIO subchart below (if enabled)
+    # -- Access key for object storage connection (should match MinIO subchart credentials if enabled)
     accessKey: "minio"
+    # -- Secret key for object storage connection (should match MinIO subchart credentials if enabled)
     secretKey: "ChangeThisPasswordForMinIO"
-    # The name of an existing k8s secret containing the object storage secret key
+    # -- Name of existing Kubernetes secret containing object storage secret key
     k8sSecretName: ""
+    # -- Key in the secret containing object storage secret key
     k8sSecretKey: "rootPassword"
-    # Bucket information
-    # Auto set to MinIO subchart service if enabled and this string is empty: "http://{{ .Release.Name }}-minio:9000"
+    # -- Object storage endpoint URL (auto-configured if MinIO subchart is enabled and this is empty)
     endpoint: ""
+    # -- Object storage bucket name for TheHive data
     bucket: "thehive"
+    # -- Object storage region
     region: "us-east-1"
+    # -- Use path-style access for object storage (required for MinIO)
     usePathAccessStyle: true
 
-  # TheHive Deployment initContainers configuration
   initContainers:
     image:
+      # -- Docker registry for init container image
       registry: docker.io
+      # -- Init container image repository
       repository: curlimages/curl
+      # -- Init container image tag
       tag: "8.17.0"
+      # -- Init container image pull policy
       pullPolicy: IfNotPresent
 
-    # Wait for Cassandra to be reachable on port 9042
     checkCassandra:
+      # -- Enable init container to wait for Cassandra readiness on port 9042
       enabled: true
 
-    # Wait for ElasticSearch "/_cluster/health" route on port 9200 to answer
     checkElasticsearch:
+      # -- Enable init container to wait for ElasticSearch readiness
```

```suggestion
      # -- Add an init container waiting for ElasticSearch to answer when requested /_cluster/health route on port 9200
```
I'd add some details on what this init container is actually doing, what do you think?